e2e/shells/bash.test.ts

```diff
@@ -34,7 +34,7 @@ testPty({
   complete: async (bash, request) => {
     const completions = (await bash.write(`\t\t`))
       .join(`\n`)
-      .replaceAll(request.trim(), ``);
+      .replaceAll(`>${request.trim()}`, ``);
```

Comment:
Should probably be `trimRight` if prefixed with `>`.

```suggestion
      .replaceAll(`>${request.trimRight()}`, ``);
```